emissary.yaml

```diff
@@ -38,13 +38,16 @@ pipeline:
       expected-commit: d25610acbea3cd0e57f924f9f6bd9df99a7e33a3
       depth: -1 # Full history is required for version determination
 
+  - runs: |
+      sed -i 's/^urllib3==2\.3\.0$/urllib3==2.5.0/' python/requirements.txt
```

Comment:
I realized after approving/auto-merging, but should we add something that will fail here when the pattern stops matching, so we know to remove it?